File: Running Sum of 1d Array.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        sum = []
        i = 1
        sum.append(nums[0])
        while i < len(nums): 
            sum.append(sum[i-1] + nums[i])
            i += 1
        return sum

File: javascript solutions/running sum of 1d array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
    let i = sum = 0;
    const out = new Array(nums.length);
    while (i < nums.length) {
        sum += nums[i];
        out[i] = sum;
        i++
    }
    return out;
};
